openmdao/test_suite/test_examples/beam_optimization/beam_group.py

```diff
@@ -29,12 +29,12 @@ def setup(self):
         force_vector = np.zeros(2 * num_nodes)
         force_vector[-2] = -1.
 
-        inputs_comp = om.IndepVarComp()
-        inputs_comp.add_output('h', shape=num_elements)
-        self.add_subsystem('inputs_comp', inputs_comp)
+        # inputs_comp = om.IndepVarComp()
```

Comment:
Are these commented out and left here intentionally?

Reply:
Oops, forgot to take that out.  Fixed.